### DISCUSSION TITLE
Can turborepo be used for a frontend monorepo-tool consisting of vue + vite applications?

### DISCUSSION THREAD

**Discussion by max-abclabs** [vercel/turborepo#236] [Help] [answered]
See title

**Comment by chris-nccpa** [comment]
@max-abclabs I'm working on this now..
https://github.com/vercel/turborepo/discussions/1502#discussioncomment-3423446

**Comment by mehulkar** [comment, marked as answer]
There shouldn't be anything preventing this. If there's something not working, please open an issue! 

I know this reply is over a year later, but if you're still feeling up for it, feel free to submit to [the examples directory](https://github.com/vercel/turbo/tree/main/examples)! cc @anthonyshew
